src/components/Content/Pursuance/PursuancePage.js

```diff
@@ -9,19 +9,71 @@ import DiscussView from './views/DiscussView';
 import ParticipantsView from './views/ParticipantsView';
 import RightPanel from '../RightPanel/RightPanel';
 import './PursuancePage.css';
+import ReactModal from 'react-modal';
 
 class PursuancePage extends Component {
-
+ constructor(){
+  super();
+  this.state={
+    modalIsOpen : false
+   }
+  this.openModal = this.openModal.bind(this);
+  this.closeModal = this.closeModal.bind(this);
+  this.handleKeydown = this.handleKeyDown.bind(this);
+ }
   componentWillMount() {
     let { setCurrentPursuance, match, currentPursuanceId } = this.props;
     currentPursuanceId = Number(match.params.pursuanceId) || currentPursuanceId;
     setCurrentPursuance(currentPursuanceId);
+    ReactModal.setAppElement('body');
+  }
+  componentWillUnmount (){
+   document.removeEventListener('keydown',this.handleKeyDown);
+  }
+  componentDidMount(){
+   this.setState({modalIsOpen : true});
+   document.addEventListener('keydown',this.handleKeyDown);
```

Comment:
@greg-han The React way of doing this is to pass `onKeyDown={this.handleKeyDown}` to the relevant component/HTML element in the `render` method.